### ISSUE TITLE
Readme feedback

### ISSUE THREAD

**Issue by addyosmani** [googlearchive/todomvc#5]
> Non-visual elements such as the router and the model are also implemented as custom elements and appear in the DOM.

Could you summarize in a line or two why these were implemented as custom elements over say, just vanilla objects?

> Since Web Components allow you to create new types of DOM elements

Link up 'Web Components' - perhaps to a primer on them.

> Polymer and its polyfills are intended to work in the latest version of evergreen browsers

It may be worth linking up 'evergreen browsers' to http://tomdale.net/2013/05/evergreen-browsers/

> https://github.com/Polymer/todomvc/blob/master/readme_template.md

This should be good to nuke now :)

**Comment by morethanreal** [comment]
Done.
